docker-compose.yml

```diff
@@ -23,6 +23,13 @@ services:
       - "14250:14250"
       - "14268:14268"
       - "16686:16686"
+    # This dependency is to insure that localstack is fully initialized and the test bucket
+    # created before docker-compose considers the stack fully up. Without this the localstack
+    # image will be labeled ok with indeterminate latency in bucket creation because the
```

Comment:
I had forgotten the nuance between `indeterminate` and `undetermined`. Thanks :)